packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -980,6 +984,18 @@ public Boolean isInfoWindowShown(@NonNull String markerId) {
     return lastSetStyleSucceeded;
   }
 
+  @Override
+  public @NonNull Boolean isAdvancedMarkersAvailable() {
```

Comment:
Can this be bool instead of Boolean if it is non null?

Reply:
Looks like it can't be done like this. This code uses Pigeon-generated code and Dart's `bool isAdvancedMarkersAvailable()` is translated into Java's `@NonNull Boolean isAdvancedMarkersAvailable();`